### PR TITLE
Updated Spring Boot version to 3.1.1

### LINKED ISSUE #89: Update Spring Boot Version (3.1.x)

### PR DESCRIPTION
**Pull request by vishalsingh2972**

### CODE REVIEW

**Comment by vishalsingh2972** [comment]
@gossie could you please review this PR!

File: app/pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>3.0.2</version>
+        <version>3.1.0</version>
```
Comment (gossie) [inline app/pom.xml:11]: There is already 3.1.1, let's use the latest and greatest.

**Review by gossie — CHANGES_REQUESTED** [review]

File: app/pom.xml (hunk 1 above)
Comment (vishalsingh2972) [reply app/pom.xml:11]: k making the changes

Comment (vishalsingh2972) [reply app/pom.xml]: Made the changes can you review the PR once

**Comment by vishalsingh2972** [comment]
made the changes you requested @gossie !

**Review by gossie — APPROVED** [review]

**Pull request merged**